nevis/_util.py

```diff
@@ -110,3 +112,141 @@ def print_result(x, y, z):
     photo = hill.photo()
     if photo:
         print('  ' + photo)
+
+
+def generate_kml(file, labels=None, trajectory=None, points=None,):
+    """
+    Generate a KML (Keyhole Markup Language, used to display geographic data
+    in an Earth browser such as Google Earth) file from given data.
+
+    Arguments:
+
+    ``file``
+        The name of the file to write to.
+    ``labels``
```

Comment:
How will these be encoded in the KML file? Can you give some hint what labels, trajectories & points will look like?